content/topics/android/intro-to-android/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Introduction to Android
+ready: true 
+---
+
+For this course we are going to be using the Android Studio IDE created by Google and the programming language Kotlin.
+
+The links below describe how to set up your development environment.
+
+## Getting started with the Android development environment
```

Comment:
This looks like it's in the wrong place: 

https://github.com/Umuzi-org/tech-department/issues/349

So basically we need to hold their hands and walk them through the process of getting up an running on a ubuntu/mint system.  And that topic should probably be mostly populated by someone who is quite familiar with Ubuntu/Mint. These resources could be usefu but it seems to lack cohesion.  (we treat docs like code: DRY and cohesive with clear intentions)